src/app/lib/initializer.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+init::run() {
```

Comment:
This isn't totally intuitive, but my intention isn't that each plugin type implements `run`, but that it implements the subcommands which it provides. (`horde up` is actually an alias for `horde run`)

```suggestion
initializer::init() {
```

Reply:
also, the plugin type should be a noun: you run `init` on your `initializer`